blueocean-web/src/main/js/init.jsx

```diff
@@ -48,6 +17,11 @@ exports.initialize = function (oncomplete) {
     const jdl = require('@jenkins-cd/design-language');
     jenkinsMods.export('jenkins-cd', 'jdl', jdl);
 
+    // Create and export a shared instance of the core
+    // js module
+    const corejs = require('@jenkins-cd/blueocean-core-js');
+    jenkinsMods.export('jenkins-cd', 'blueocean-core-js', corejs);
+    
```

Comment:
@cliffmeyers is also doing this in one of his PRs. Once #416 gets merged [we'll specify this stuff on the bundle generation](https://github.com/jenkinsci/blueocean-plugin/blob/3944ebeca26a64b9b6dc3eb1c50c1057305ec0c1/blueocean-web/gulpfile.js#L39-L43).